autoreject/autoreject.py

```diff
@@ -1392,3 +1417,66 @@ def plot_epochs(self, epochs, scalings=None, title=''):
             epochs=epochs,
             epoch_colors=epoch_colors, scalings=scalings,
             title='')
+
+    def drop_epochs_with_adjacent_channel_interpolation(
+            self, ch_adjacency, ch_names):
```

Comment:
I would make this a function, not method.

Reply:
Why not operate on the reject log? This seems cleaner.

Reply:
hold on ... I am not done with review. I think I accidentally forgot to "start a review" so you start getting comments until I am finished. Still forming my thoughts :)